src/metadata/persistence/favorites.clj

```diff
@@ -31,10 +37,10 @@
    Returns:
      A base selection query."
   [user target-types]
-  (-> (select* :favorites)
-      (fields :target_id)
-      (where {:target_type [in (map db/->enum-val target-types)]
-              :owner_id    user})))
+  (-> (h/select :target_id)
+      (h/from (t "favorites"))
+      (h/where [:in :target_type [:inline target-types]]
```

Comment:
I'm... actually not sure. I did this namespace first, so it could be that I actually have it wrong in the other namespace. I think they might work as alternates though -- `jtypes/as-other` will mark them correctly to be included as bound parameters, while `:inline` will inline them in the expression rather than using bind parameters at all. I think that since in raw SQL without bind parameters you can basically treat enums like strings, it might just work out the same.

I think this inline version is a bit cleaner, so when I get a chance to get back to this and test it out, I might just use that everywhere I can.